express/parsers/molecule.py

```diff
@@ -0,0 +1,113 @@
+from io import StringIO
+import os
+from ase.io import read, write
+import ase
+import rdkit
+from rdkit import Chem
+import logging
+
+class MoleculeParser():
+    """
+    Molecule parser class.
+
+    Args:
+        args (list): args passed to the parser.
+        kwargs (dict): kwargs passed to the parser.
+            structure_string (str): structure string.
+            structure_format (str): structure format, poscar or espresso-in.
+    """
+
+    def __init__(self, structure_string):
+        self.structure_string = structure_string
+        self.inchi_run = self.try_openbabel_import()
```

Comment:
for boolean flags, which this is, let's use `is_*` in the name pls - `self.is_using_inchi`